net/remote/remote-node.sh

```diff
@@ -28,13 +28,6 @@ gpuMode="${19:-auto}"
 GEOLOCATION_API_KEY="${20}"
 set +x
 
-# Use a very large stake (relative to the default multinode-demo/ stake of 42)
-# for the testnet validators setup by net/.  This make it less likely that
-# low-staked ephemeral validator a random user may attach to testnet will cause
-# trouble
-#
-# Ref: https://github.com/solana-labs/solana/issues/3798
-stake=${internalNodesStakeLamports:=424243}
```

Comment:
We still need some default value for `internalNodesStakeLamports` . It is an optional value in the calling scripts.

Reply:
I saw 1SOL is the default

Reply:
It is set to 1 SOL for TdS network config, but is otherwise unset, so we will end up with the 0.5 SOL in the delegate-stake.sh script default.